util/test_timeline.py

```diff
@@ -424,12 +424,12 @@ def timeline_file(filename):
     # Allow for just specifying the base filename, e.g. "o12s.txt" or "o12s"
     if not os.path.exists(filename):
         for root, dirs, files in os.walk(data_path):
-          if filename in files:
-            filename = os.path.join(root, filename)
-            break
-          if '%s.txt' % filename in files:
-            filename = os.path.join(root, '%s.txt' % filename)
-            break
+            if filename in files:
```

Comment:
Huh! I would have thought that pylint would have been mad at this.  Thanks for fixing it.

Reply:
I wouldn't normally have done that in this sequence of requests, but given that I already had to make a mostly-cosmetic change, I figured it was reasonable to toss this in as well.